### PR TITLE
drop roblox clients

### LINKED ISSUE #220062: grapejuice can be broken by new wine versions
### Describe the bug
When a new version of wine is packaged, the version pointed to by the config (.config/brinkervii/grapejuice/user_settings.json) (a store link) becomes invalid. At best, this will use an old wine version, and at worst will cause the app to fail to launch.

### Steps To Reproduce
1. Install old wine
2. Generate initial config (`grapejuice first-time-setup`)
3. Update wine
4. Run `grapejuice app`
5. Observe

### Expected behavior
The app should not crash.

### Screenshots
N/A

### Additional context
Unsure how Nix should handle this one. The *best* solution is to update the config file (which conforms to xdg), however that may be out of scope. A warning / doc would also help.

### Notify maintainers
@artturin @helium

### Metadata
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.91, NixOS, 22.11 (Raccoon), 22.11.20230204.8e82401`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.1`
 - channels(root): `"home-manager-22.05.tar.gz, nixos-22.11, nixos-unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

```console
amy:~/ $ nix-shell -p nix-info --run "nix-info -m"                                          [21:56:17]
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.91, NixOS, 22.11 (Raccoon), 22.11.20230204.8e82401`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.1`
 - channels(root): `"home-manager-22.05.tar.gz, nixos-22.11, nixos-unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

amy:~/ $
```

### LINKED ISSUE #264173: Grapejuice: RobloxPlayer always installing
So, whenever I open Player, it starts installing Roblox, finishes, and does nothing. Even when launched from the website, starts installing, finishes, and most likely just exits. Logs show nothing important. However, when trying open Studio, the error saying "RobloxStudioBeta.exe not found".

1. Download the grapejuice package (just write grapejuice in the with pkgs section)
2. Download wine-wayland package (not sure if it is required, however I did)
2. Open Grapejuice (I used rofi (drun), but I suppose $ `grapejuice app` would work)
3. In the message box that pops out (from wine), press "Install" to install the wine-mono thingy
4. Either press the Open RobloxPlayer button, or go into Player, Initialize and Install Roblox, the behavior is the same
5. Enjoy Roblox installing and doing nothing...

I expected RobloxPlayer to actually open, not install and exit...

```console
[xceptional@nixos:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.58, NixOS, 23.11 (Tapir), 23.11pre537656.7c9cc5a6e5d3`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.17.1`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

### LINKED ISSUE #282315: Studio not working in Vinegar?
### Describe the bug
The "Studio" option is not working in Vinegar. The application does download, the application does launch. However, once it is open, the window is almost unusable.

### Steps To Reproduce
1. Download vinegar from nixpkgs (unstable)
2. Run `vinegar studio` in the terminal
3. Enjoy the mess in front of your eyes...

### Expected behavior
So, instead of getting the usual Studio look, I see the username, question cicrle, notification bell, and "FILE" buttons (all of the mentioned buttons do work, so I AM able to click on them and get a popup, as well as use the options from there, as well as hover effects). Everything else looks like a badly margined css page. (Replaced the actual username with "username" in the picture below, might not be perfectly aligned, just to make no misconceptions, the button IS properly aligned in the original image!)

### Screenshots
2024-01-20-130051_hyprshot

### Notify maintainers
@nyabinary

### Metadata
```console
[xceptional@nixos:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.72, NixOS, 24.05 (Uakari), 24.05.20240115.c3e128f`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

### PR DESCRIPTION
**Pull request by eclairevoyant**
## Description of changes

In light of the (old) news that https://www.gamingonlinux.com/2024/03/game-over-for-roblox-on-linux-steam-deck-as-its-now-blocked/

- **vinegar: remove**
  - closes #282315
- **grapejuice: remove**
  - closes #264173
  - closes #220062

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by matteo-pacini — APPROVED** [review]
LGTM

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -605,6 +605,7 @@ mapAliases ({
   gradle_5 = throw "gradle_5 has been removed because it's no longer being updated"; # Added 2023-01-17
   grafana_reporter = grafana-reporter; # Added 2024-06-09
   gr-ais = throw "'gr-ais' has been renamed to/replaced by 'gnuradio3_7.pkgs.ais'"; # Converted to throw 2023-09-10
+  grapefruit = throw "'grapefruit' was removed due to being blocked by Roblox, rendering the package ineffective"; # Added 2024-08-23
```
Comment (SuperSandro2000) [inline pkgs/top-level/aliases.nix:608]:
```suggestion
  grapefruit = throw "'grapefruit' was removed due to being blocked by Roblox, rendering the package useless"; # Added 2024-08-23
```

File: pkgs/top-level/aliases.nix [hunk 2]
```diff
@@ -1547,6 +1548,7 @@ mapAliases ({
   vikunja-frontend = throw "'vikunja-frontend' has been replaced by 'vikunja'"; # Added 2024-02-19
   vimHugeX = vim-full; # Added 2022-12-04
   vim_configurable = vim-full; # Added 2022-12-04
+  vinegar = throw "'vinegar' was removed due to being blocked by Roblox, rendering the package ineffective"; # Added 2024-08-23
```
Comment (SuperSandro2000) [inline pkgs/top-level/aliases.nix:1551]:
```suggestion
  vinegar = throw "'vinegar' was removed due to being blocked by Roblox, rendering the package useless"; # Added 2024-08-23
```

**Comment by SystematicError** [comment]
Only roblox player is blocked on wine, roblox studio still works and grapejuice / vinegar are really useful for running those

**Comment by jaschutte** [comment]
Like the comment above said, Vinegar and Grapejuice still work wonders for developing in Roblox Studio. They still serve a valuable purpose and I would like to see this reverted.

**Comment by HeitorAugustoLN** [comment]
> Like the comment above said, Vinegar and Grapejuice still work wonders for developing in Roblox Studio. They still serve a valuable purpose and I would like to see this reverted.

Same.

**Comment by Babakinha** [comment]
Yes please, rever this -.-

**Pull request merged**